src/main/features/ccj/views/check-and-send.njk

```diff
@@ -11,14 +12,15 @@
     {{ csrfProtection(csrf) }}
 
     {{ tableStart("theirDetails", "Their details") }}
-    {{ row('Full name:', user.claim.claimData.defendant.name) }}
```

Comment:
The only issue I spotted is that 'Change' link is present in 'Name', 'Business name' or 'Contact person' rows. Need to reimplement it a bit.